src/ES6/Modules.js

```diff
@@ -0,0 +1,16 @@
+// const getName = () => {
+//   return 'Simone';
+// };
+
+// exports.getName = getName;
+
+//↑ This example will work because it use common JS
+
+
+// ↓ This example will  not work if you use common JS so you need to  install typescript
+
+export default class User { // just add "default"
+  constructor(name) {
+    this.name = name;
+  }
+}
```

Comment:
How about the non-default export and import example?

Reply:
I will put them in.